applications/StructuralMechanicsApplication/structural_mechanics_application.cpp

```diff
@@ -75,6 +75,8 @@ KratosStructuralMechanicsApplication::KratosStructuralMechanicsApplication()
       // Adding the membrane element
       mPreStressMembraneElement3D3N(0, Element::GeometryType::Pointer(new Triangle3D3<NodeType >(Element::GeometryType::PointsArrayType(3)))),
       mPreStressMembraneElement3D4N(0, Element::GeometryType::Pointer(new Quadrilateral3D4<NodeType >(Element::GeometryType::PointsArrayType(4)))),
+      mMembraneElement3D4N(0, Element::GeometryType::Pointer(new Quadrilateral3D4<NodeType >(Element::GeometryType::PointsArrayType(4)))),
+      mMembraneElement3D3N(0, Element::GeometryType::Pointer(new Triangle3D3<NodeType >(Element::GeometryType::PointsArrayType(3)))),
```

Comment:
Only 3D?

Reply:
does a 2D membrane make sense (having it in Kratos)?
Isn't that a "Platte"?

Reply:
we could do this. A 2D membrane would still be a membrane but in 2D :D
but I dont see any use in this.

Reply:
a 2D membrane is like the normal TotalLagrangianElement unless you do something to improve the membrane behaviour